### DISCUSSION TITLE
How to compute the Laplacian matrix of a graph with self-loops using 'get_laplacian' function ?

### DISCUSSION THREAD

**Discussion by Nassiming** [pyg-team/pytorch_geometric#7047] [Q&A] [answered]
It seems that `get_laplacian` will automatically remove self-loops in the graph, so how can I to compute the Laplacian matrix of a graph with self loops using it? Or do I need to define my own function?

**Comment by rusty1s** [comment, marked as answer]
That is correct. I am open to make the removal of self-loops optional. If that sounds interesting to you, please feel free to submit a PR.
